shared/profile/edit-avatar-placeholder/container.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import EditAvatar from '.'
+import {connect, type TypedState} from '../../util/container'
+import {navigateUp} from '../../actions/route-tree'
+
+const mapStateToProps = (state: TypedState) => {
+  const username = state.config.username
+  if (!username) {
```

Comment:
maybe better to not throw. i know we do this in places but i think its more dangerous, if no username i'd just return mostly blank stuff and have it not render

Reply:
well, actually this is old so we can keep it, just for future reference

Reply:
Should we get an issue in JIRA to clean this up?